dep_check/use_cases/interfaces.py

```diff
@@ -16,6 +16,7 @@ class Configuration:
     dependency_rules: DependencyRules = field(default_factory=dict)
     local_init: bool = False
     error_on_unused: bool = False
+    check_unused: bool = True
```

Comment:
isn't `error_on_unused` useless now ?

Reply:
You may want the warnings but not the error. Both could probably be factored into `--unused=error|warning|ignore` but that's another story

Reply:
`--unused=error|warning|ignore` :+1: good idea